package.json

```diff
@@ -11,10 +11,12 @@
   "private": true,
   "dependencies": {
     "@angular/animations": "~12.2.0",
+    "@angular/cdk": "^12.2.12",
```

Comment:
Are you sure you want to use the ^ update type? It doesn't match the update types for the rest of your dependencies

Reply:
Fixed it